### ISSUE TITLE
how does the inference model trained?

### ISSUE THREAD

**Issue by 853498747** [Gaoyiminggithub/Graphonomy#2]
Awesome job !does the inference model use transfer? I want to change the body parts into the same labe to extract person contours，will the training effect be better？ The inferrence model didn't work well for my some small and low definition images,(I just change scale list into 2). 
        There is only one person in my picture ,should i train ATR or LIP use deeplabv3 directly ,can you give some advice or train step ? Thankyou！

**Comment by Gaoyiminggithub** [comment]
1. inference model uses transfer.
2. I think the training may be better.
3. If the image resolution is so small (e.g. 64 x 64), the results may be worse. ( I don't understand `(I just change scale list into 2)`)
4. You can use our inference model to deal with the setting of only one person in your picture. If you want to train ATR or LIP using deeplabv3, you can use the [deeplabv3+ model directly](https://github.com/Gaoyiminggithub/Graphonomy/blob/master/networks/deeplab_xception.py#L394).
You may need to download the LIP dataset, change the dataloader( you can refer the [cihp.py](https://github.com/Gaoyiminggithub/Graphonomy/blob/master/dataloaders/cihp.py) ), and then change the [model](https://github.com/Gaoyiminggithub/Graphonomy/blob/master/exp/transfer/train_cihp_from_pascal.py#L179) and [forward](https://github.com/Gaoyiminggithub/Graphonomy/blob/master/exp/transfer/train_cihp_from_pascal.py#L275) in the train.py. 
Maybe it can help you.

**Comment by 853498747** [comment]
Thankyou for your advice
1  . Scale_list is the variable in inference.py in line 110.
 I change the default list into ［1.25，1.5，1.75］to extract my 100*200 picture is better than default  ，if retrain maybe i should change the training scale ?
but my picture(person detection box) is various size 100×200 ~300×700,I'm also confuse about how to resize..
2. If i change body parts into one label and run train_transfer_cihp.sh ,can you update pascal pretrained model.

**Comment by Gaoyiminggithub** [comment]
1. Oh, I understand. If the image is 100*200, you could try the list with larger scale (e.g., [1.25,1.5,1.75,2,2.25]). You can change the training random scale in [here](https://github.com/Gaoyiminggithub/Graphonomy/blob/master/dataloaders/custom_transforms.py#L455).
2. I will update the other models as soon as possible after the ddl on 23 May TAT. So sorry about this.

**Comment by 853498747** [comment]
thank u,best wishes
